lib/resources/users.js

```diff
@@ -80,21 +81,29 @@ module.exports = (service, endpoint) => {
         .then(() => user))));
 
   // TODO: infosec debate around 404 vs 403 if insufficient privs but record DNE.
-  service.patch('/users/:id', endpoint(({ User }, { params, body, auth }) =>
+  service.patch('/users/:id', endpoint(({ User, mail }, { params, body, auth }) =>
     User.getByActorId(params.id)
       .then(getOrNotFound)
       .then((user) => auth.canOrReject('user.update', user.actor)
-        .then(() => user.with(User.fromApi(body)).update()))));
+        .then(() => user.with(User.fromApi(body)).update())
+        .then((response) => {
```

Comment:
i'd prefer this to be named `result` or `savedUser` than `response`, since that name in this context implies to me the HTTP/Express response object.

Reply:
Sounds good, changed to result in latest commit.